front/lib/api/assistant/configuration.ts

```diff
@@ -1490,15 +1491,10 @@ async function _createAgentDataSourcesConfigData(
   // Then we get do one findAllQuery per workspaceId, in a Promise.all
   const getDataSourcesQueries = dsNamesPerWorkspaceId.map(
     ({ workspaceId, dataSourceNames }) => {
-      return DataSource.findAll({
-        where: {
-          workspaceId,
-          name: {
-            [Op.in]: dataSourceNames,
-          },
-        },
-        transaction: t,
-      });
+      return DataSourceResource.listByWorkspaceIdAndNames(
+        workspaceId,
```

Comment:
Either we pass `Authenticator` if we manage to remove the "multiple workspace ids" or we pass a `LightWorkspaceType` but no ModelId.